Create ui-c8y-1019-13-5-fixed-an-issue-in-password-reset-form-which-prevented-user-from.md (#1853)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-13-5-fixed-an-issue-in-password-reset-form-which-prevented-user-from.md

---
date: ""

> title: Fixed password reset form preventing users from setting a new password

Reviewer: ```suggestion
title: Users can reset their password properly
```

> title: Fixed an issue in password reset form which prevented user from setting a new password [GRAFT][release/cd] (#5965)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed password reset form preventing users from setting a new password
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58234
version: 1019.13.5
---

> Fixed an issue in password reset form which prevented user from setting a new password [GRAFT][release/cd] (#5965)

Author: The Claude AI suggested to use the following body:
```suggestion
In some cases, users were unable to set a new password using the password reset form. This issue has been resolved and users can now successfully reset their password as expected. Existing users who encountered this problem will now be able to reset their password without any errors.
```
